chain/chain/src/store_validator/validate.rs

```diff
@@ -295,6 +295,34 @@ pub(crate) fn chunk_indexed_by_height_created(
     Ok(())
 }
 
+pub(crate) fn chunk_tx_exists(
+    sv: &mut StoreValidator,
+    _chunk_hash: &ChunkHash,
+    shard_chunk: &ShardChunk,
+) -> Result<(), StoreValidatorError> {
+    for tx in shard_chunk.transactions.iter() {
+        let tx_hash = tx.get_hash();
+        unwrap_or_err_db!(
+            sv.store.get_ser::<SignedTransaction>(ColTransactions, &tx_hash.as_ref()),
+            "Can't get Tx from storage for Tx Hash {:?}",
+            tx_hash
+        );
+    }
+    Ok(())
+}
+
+pub(crate) fn chunk_increase_tx_refcount(
+    sv: &mut StoreValidator,
+    _chunk_hash: &ChunkHash,
+    shard_chunk: &ShardChunk,
+) -> Result<(), StoreValidatorError> {
+    for tx in shard_chunk.transactions.iter() {
+        sv.inner.tx_refcount.entry(tx.get_hash()).and_modify(|x| *x += 1).or_insert(1);
+    }
+    sv.inner.is_tx_refcount_calculated = true;
```

Comment:
What does this variable tracks?
Seems like it gets set to `true`  after just one chunk being processed?

Reply:
Discussed offline.